app/src/main/kotlin/com/wire/android/ui/common/OnDropDownIconButton.kt

```diff
@@ -0,0 +1,22 @@
+package com.wire.android.ui.common
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowDropDown
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+
+
+@Composable
+fun OnDropDownIconButton(onDropDownClick: () -> Unit) {
+    IconButton(
+        onClick = onDropDownClick
+    ) {
+        Icon(
+            imageVector = Icons.Filled.ArrowDropDown,
+            contentDescription = "",
+            tint = MaterialTheme.colorScheme.onBackground
```

Comment:
I think we shouldn't set the tint, `TopAppBar` handles it by itself so you should rather define colors directly for TopAppBar using `TopAppBarColors` (`actionIconContentColor`, `navigationIconContentColor` `titleContentColor`)
this should apply to all icons in the TopAppBar

Reply:
true, wasn't aware of that